measures/ResStockArguments/measure.rb

```diff
@@ -517,12 +517,28 @@ def run(model, runner, user_arguments)
     args[:hvac_control_cooling_weekend_setpoint] = weekend_cooling_setpoints.join(', ')
 
     # Seasons
-    if args[:use_auto_heating_season] && args[:hvac_control_heating_season_period].to_s == Constants.Auto
+    if args[:use_auto_heating_season]
       args[:hvac_control_heating_season_period] = HPXML::BuildingAmerica
+    else
+      if args[:hvac_control_heating_season_period] == 'Unavailable 1 Day'
+        args[:hvac_control_heating_season_period] = 'Jan 16 - Jan 14'
+      elsif args[:hvac_control_heating_season_period] == 'Unavailable 1 Week'
+        args[:hvac_control_heating_season_period] = 'Jan 18 - Jan 10'
+      elsif args[:hvac_control_heating_season_period] == 'Unavailable 1 Month'
+        args[:hvac_control_heating_season_period] = 'Jan 20 - Dec 20'
+      end
```

Comment:
Consider name change from "Heating Season.tsv" to "Heating Unavailable Days.tsv" or "Heating Unavailable Duration.tsv". Are we doing this for cooling, too?

How do we determine "seasons" from which to randomly select an N day unavailable period? We could use the Building America season, and then randomly (?) choose a period falling within this season.

We should report start date, end date, etc. Not only for QAQC but for generally understanding why there'd be no heating/cooling during a portion of the year.

@rajeee @afontani